platform/linux-generic/odp_ipsec.c

```diff
@@ -1889,6 +1892,25 @@ int odp_ipsec_out_inline(const odp_packet_t pkt_in[], int num_in,
 			ODP_ASSERT(ODP_IPSEC_SA_INVALID != sa);
 		}
 
+		hdr_len = inline_param[in_pkt].outer_hdr.len;
+		ptr = inline_param[in_pkt].outer_hdr.ptr;
+
+		if (!ptr) {
+			uint32_t l2_offset = odp_packet_l2_offset(pkt);
+
+			ODP_ASSERT(hdr_len == odp_packet_l3_offset(pkt) - l2_offset);
```

Comment:
Are there 2 blank lines here?

Reply:
I am not sure what you mean. There is a blank line both before and after the assert, if that is what you mean.

Reply:
It looked like there were two blank lines after line 1901. But looking again, I must have gotten it wrong.